### PR TITLE
[CRS/202412] changes section from FPWD

### PR DESCRIPTION
**Pull request by himorin**
as title

not including consideration sections: 251, 247, 219, 216, 207

***
<a href="https://pr-preview.s3.amazonaws.com/himorin/w3c-compute-pressure/pull/306.html" title="Last updated on Dec 11, 2024, 9:51 AM UTC (ade5b1c)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/compute-pressure/306/1726daf...himorin:ade5b1c.html" title="Last updated on Dec 11, 2024, 9:51 AM UTC (ade5b1c)">Diff</a>

### CODE REVIEW

**Review by anssiko — APPROVED** [review]
Thank you! I note the commit message titles are directly reused here without edits. An improvement might be to adjust wording in a few places, but I'm fine with this too.

@kenchris to check this looks good from an editor's perspective.

**Review by kenchris — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -2540,9 +2540,45 @@ <h2>
...
 
-<section id="index" class="appendix">
+<section id="substantive-changes-summary" class="appendix informative">
+  <h2>Summary of substantive changes</h2>
+  <section id="substantive-changes-summary-fpwd">
+    <h3>Changes since the <a href="https://www.w3.org/TR/2022/WD-compute-pressure-20221220/">First Public Working Draft (2022/12/20)</a></h3>
+    <ul>
+      <li><a href="https://github.com/w3c/compute-pressure/pull/302">Remove the limiting of rate as it is replace by obfuscation (PR #302)</a></li>
+      <li><a href="https://github.com/w3c/compute-pressure/pull/300">Move at-risk to experimental on PressureSource/thermals (PR #300)</a></li>
+      <li><a href="https://github.com/w3c/compute-pressure/pull/299">Make ownContributionEstimate experimental (PR #299)</a></li>
```
Comment (kenchris) [inline index.html:2550]: This entry and the next are not visible in the non-editors version, so can be removed

File: index.html [hunk 2]
```diff
@@ -2540,9 +2540,45 @@ <h2>
...
+      <li><a href="https://github.com/w3c/compute-pressure/pull/298">Add support for ownContributionEstimate (PR #298)</a></li>
```
Comment (kenchris) [inline index.html:2551]: And this one

**Review by kenchris — CHANGES_REQUESTED** [review]
Let's remove the two entries not visible in the non-editors version

**Review by arskama — APPROVED** [review]
LGTM

**Review by anssiko — APPROVED** [review]

**Pull request merged**